pkg/vm/evm_prompts.go

```diff
@@ -174,16 +175,20 @@ func PromptSubnetEVMGenesisParams(
 
 	if sc.PoA() {
 		params.UsePoAValidatorManager = true
-		params.initialTokenAllocation[common.HexToAddress(sc.PoAValidatorManagerOwner)] = core.GenesisAccount{
+		params.initialTokenAllocation[common.HexToAddress(sc.ValidatorManagerOwner)] = core.GenesisAccount{
 			Balance: defaultPoAOwnerBalance,
 		}
 	}
 
 	if sc.PoS() {
+		params.UsePoSValidatorManager = true
+
 		params.enableNativeMinterPrecompile = true
-		params.nativeMinterPrecompileAllowList.EnabledAddresses = []common.Address{
-			common.HexToAddress(validatormanager.ValidatorContractAddress),
-		}
+		params.nativeMinterPrecompileAllowList.AdminAddresses = append(
```

Comment:
why admin? is not enough to use enabled?

Reply:
changed to enabled